<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LabelingDialog</class>
 <widget class="QDialog" name="LabelingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>391</width>
    <height>774</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_14">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_AddLabel">
      <attribute name="title">
       <string>Add Label</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_11">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Input Text:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="le_InputText"/>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_13">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Location:</string>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLineEdit" name="le_Lon">
                  <property name="placeholderText">
                   <string>Lon</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_Lat">
                  <property name="placeholderText">
                   <string>Lat</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="pb_LocationPicker">
                  <property name="maximumSize">
                   <size>
                    <width>30</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../Resources.qrc">
                    <normaloff>:/new/prefix1/icons/drawTools/LocationMarker.png</normaloff>:/new/prefix1/icons/drawTools/LocationMarker.png</iconset>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_TextStyle_2">
         <property name="title">
          <string>Text Style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QGridLayout" name="gridLayout_6">
            <item row="2" column="0">
             <widget class="QLabel" name="lblBuffer_3">
              <property name="text">
               <string>Color:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="sb_fontSize">
              <property name="minimum">
               <number>6</number>
              </property>
              <property name="maximum">
               <number>36</number>
              </property>
              <property name="value">
               <number>10</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="lblBuffer_2">
              <property name="text">
               <string>Buffer:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cb_Font"/>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLineEdit" name="le_fillColor"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_fillColor">
                <property name="text">
                 <string>Pick a color</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="lblBuffer_4">
              <property name="text">
               <string>Allignment:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lblFont_2">
              <property name="text">
               <string>Font:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLineEdit" name="le_haloColor"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_haloColor">
                <property name="text">
                 <string>Pick a color</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblSize_2">
              <property name="text">
               <string>Size:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="cb_textAlignment"/>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>BufferWidth::</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QSpinBox" name="sp_haloWidth"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb_enableLabelBox">
         <property name="text">
          <string>Enable Label Box</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="GB_LabelBox">
         <property name="title">
          <string>Label Box Style:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <layout class="QGridLayout" name="gridLayout_7">
            <item row="0" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLineEdit" name="le_LabelBoxFillColor"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_LabelBoxFillColor">
                <property name="text">
                 <string>Pick a color</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="lblBuffer_6">
              <property name="text">
               <string>Border:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLineEdit" name="le_LabelBoxBorderColor"/>
              </item>
              <item>
               <widget class="QPushButton" name="pb_LabelBoxFillColor_2">
                <property name="text">
                 <string>Pick a color</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="lblBuffer_5">
              <property name="text">
               <string>Fill :</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="lblBuffer_7">
              <property name="text">
               <string>Border Width:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="sb_LabelBoxBorderWidth"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Visbility scale:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cb_VisibilityScale">
           <item>
            <property name="text">
             <string>On Earth</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>On Zoom</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <widget class="QPushButton" name="pb_GetStyle">
           <property name="text">
            <string>GetStyle</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_AddText">
           <property name="text">
            <string>Add</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_reset">
           <property name="text">
            <string>reset</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pb_cancel">
           <property name="text">
            <string>cancel</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
